Makefile

```diff
@@ -460,6 +460,10 @@ $(eval $(call build-package,dataplaneapi,2.7.2-r0))
 $(eval $(call build-package,wasmtime,6.0.0-r0))
 $(eval $(call build-package,poetry,1.3.2-r0))
 
+
+$(eval $(call build-package,zot,1.4.3-r0))
+
```

Comment:
Any reason for this not being together with the other? and those extra lines?

Reply:
I think this may be due to a miscommunication.

When you add a package to the Makefile, it can be useful to add the new line _not_ at the bottom of the list -- if everyone does this, there will be merge conflicts on this last line. At the same time, it's a good idea to put it _near_ the end of the bottom, to ensure any build dependencies are built before the new package.

I think when I'd suggested this before I didn't communicate it clearly: the package should be added 5-10 lines from the bottom of the list, not spaced after the list.

So something like:

```
...
$(eval $(call build-package,dataplaneapi,2.7.2-r0))
$(eval $(call build-package,wasmtime,6.0.0-r0))
<---- put your new package here ---->
$(eval $(call build-package,poetry,1.3.2-r0))
$(eval $(call build-package,zot,1.4.3-r0))
$(eval $(call build-package,terraform,1.3.9-r0))
```